spec/core/string/shared/chars.rb

```diff
@@ -57,10 +57,12 @@
       [0xAD].pack('C').force_encoding('BINARY'),
       [0xA2].pack('C').force_encoding('BINARY')
     ]
-    #s.force_encoding('SJIS').send(@method).to_a.should == [
-      #[0xF0,0xA4].pack('CC').force_encoding('SJIS'),
-      #[0xAD].pack('C').force_encoding('SJIS'),
-      #[0xA2].pack('C').force_encoding('SJIS')
-    #]
+    NATFIXME 'Implement SJIS encoding', exception: ArgumentError, message: 'unknown encoding name - "SJIS"' do
```

Comment:
To save everybody else the need to look it up: `SJIS` is not the same as `Shift-JIS`, but an alias for `Windows-31J`